### ISSUE TITLE
Explorer help needs to be updated.

### ISSUE THREAD

**Issue by SimonGoring** [NeotomaDB/Explorer#5]
The help documentation or Neotoma Explorer needs to be updated.

Currently the help documentation for Tilia & the Database are both archived & hosted through [readthedocs](http://readthedocs.io).

* [Neotoma Database manual](neotoma-manual.readthedocs.io/en/latest/) & [Neotoma Manual GitHub Repository](https://github.com/SimonGoring/neotomadb-manual/)
* [Tilia Manual](http://tilia-manual.readthedocs.io/en/latest/) & [Tilia Manual GitHub Repository](https://github.com/SimonGoring/tilia-manual)

This may be a broader "Documentation" question though.

**Comment by spatialit** [comment]
I also added this to the closed thread by accident. 

I looked at both DokuWiki and HelpNDoc. DokuWiki, or something similar, is what we want. It allows distributed editing and includes access control. Seems like both have a "source" version and the generated documents. I asked Brian where the source is but haven't heard back.

**Comment by spatialit** [comment]
Yes, the API should generate it's own documentation. DokuWiki looks good for documentation with authors.
